osu.Game/Screens/BackgroundScreen.cs

```diff
@@ -45,7 +45,6 @@ protected override bool OnKeyDown(KeyDownEvent e)
         protected override void Update()
         {
             base.Update();
-            Scale = new Vector2(1 + x_movement_amount / DrawSize.X * 2);
```

Comment:
I'm not sure why this was removed with no explanation. This update method is now a no-op.